ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -177,159 +177,56 @@ export class WorkflowDetails extends React.Component<RouteComponentProps<any>, W
         );
     }
 
+    private confirmAction(title: string): void {
+        if (!confirm(`Are you sure you want to ${title} this workflow?`)) {
+            return;
+        }
+        return;
+    }
```

Comment:
Is this function correct? It doesn't seem the `if` statement has any effect here

Reply:
I think this is correct, I tried it without the `if` and it didn't work (the confirm dialogue didn't pop up)

Reply:
I meant more that it doesn't matter if `confirm` returns `true` or `false`, the result doesn't chagne